.ci/Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent {label "clx-host-21514||clx-host-21518"}
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr:'10'))
+        timeout(time: 3, unit: 'MINUTES')
+        disableConcurrentBuilds()
+    }
+
+    environment {
+        SRC_ROOT = "${env.WORKSPACE}"
```

Comment:
why needed in env section? if it is already part of ${env.xxx} ? IMO - not needed as it is already there.

Reply:
This's mainly for naming convenience - IMHO WORKSPACE is too generic name. I'm OK to remove the additional variables to simplify the environment section.